woocommerce/woocommerce-ios
# Order Detail: Fixed dark mode appearance in some cells

Fixes #1389 

The scope of the PR is limited to just making the Order Detail screen usable in Dark mode when building the codebase with the release version of Xcode 11.

## Testing
- Activate dark mode
- Navigate to an Order Detail
- Check that everything is visually ok (readable, and without dark background)

## Screenshots

| Before             |  After |
:-------------------------:|:-------------------------:
![67297081-b273c680-f4e9-11e9-883b-983cb3d46d36](https://user-images.githubusercontent.com/495617/68483828-891e9e80-023c-11ea-8b60-45e0a3401749.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2019-11-08 at 15 20 50](https://user-images.githubusercontent.com/495617/68483849-8fad1600-023c-11ea-990c-4b0625dd5ed0.png)




Update release notes:
